src/NUnitFramework/framework/Attributes/IgnoreAttribute.cs

```diff
@@ -85,8 +86,7 @@ public void ApplyToTest(Test test)
                     if (_untilDate.Value > DateTime.Now)
                     {
                         test.RunState = RunState.Ignored;
-                        string reason = string.Format("Ignoring until {0}. {1}", _untilDate.Value.ToString("u"), _reason);
-                        test.Properties.Set(PropertyNames.SkipReason, reason);
+                        test.AddIgnoreUntilReason(_untilDate.Value, _reason);
```

Comment:
@jnm2 I couldn't include the `RunState` in the de-duplication easily. The interface is `get` only and the derived classes have a wide range of private and public sets.

Reply:
Thanks, I didn't realize that at the time.